rental_base/views/menu_view.xml

```diff
@@ -84,78 +84,90 @@
     <menuitem
         name="Customers"
         id="menu_top_customer"
+        groups="sales_team.group_sale_salesman"
         parent="menu_rental_root"
         sequence="10"
     />
         <menuitem
         name="Customers"
         id="menu_customers"
+        groups="sales_team.group_sale_salesman"
```

Comment:
Also change this to base.group_user because a normal user can also see the vendors, why not the customers?